src/components/Tables/TableFilterBar.tsx

```diff
@@ -48,15 +48,17 @@ const TableFilterBar: React.FC<Props> = (props) => {
                 .filter(([key]) => props.filterKeys?.includes(key) ?? true)
                 .map(([key, value]) => {
                     return (
-                        <StyledFilterBar
-                            key={key}
-                            type="text"
-                            value={props.filterText[key] ?? ""}
-                            placeholder={`Filter by ${value}`}
-                            onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
-                                props.onFilter(event, key)
-                            }
-                        />
+                        <>
```

Comment:
Why does this need to be in a fragment?

Reply:
I had the same question when I was rebasing the changes from the new theming changes. Must just be an artefact from previous changes but I'll remove it just now